src/routing/error_cause.rs

```diff
@@ -141,14 +157,11 @@ impl IntoResponse for ErrorCause {
     #[cfg(feature = "debug")]
     fn into_response(self) -> Response {
         let client_facing_error: ErrorClientFacing = (&self).into();
+        let error_data = client_facing_error.page_data();
 
-        let body = format!(
-            "error: {}\ndetails:\n{}",
-            self,
-            self.details().unwrap_or_default()
-        );
+        let body = format!("error: {}\ndetails:\n{}", self, error_data.description);
```

Comment:
If the `description` field is just for this use case (with `debug` feature) - probably it makes not much sense. You can just spit HTML here, no big deal.

Reply:
It is not just there, but in general used for any error response that is sent out for non HTTP gateway requests. I would leave it for now.